gpuarray/include/gpuarray/buffer_blas.h

```diff
@@ -1,12 +1,12 @@
-#ifndef GPUARRAY_BUFFER_BLAS_H
-#define GPUARRAY_BUFFER_BLAS_H
+#ifndef LIBGPUARRAY_GPUARRAY_BUFFER_BLAS_H
+#define LIBGPUARRAY_GPUARRAY_BUFFER_BLAS_H
```

Comment:
Google style advises <project_name>_<directories>_<file name> for a header guard. I also feel that this is redundant here, but prefixing LIBGPUARRAY to other guards (e.g. cache.h) is beneficial I think.